File: java/src/main/java/com/fengtian/qianer/atcoder/abc/abc156/ABC_156_C.java
package com.fengtian.qianer.atcoder.abc.abc156;

import java.util.Arrays;
import java.util.Scanner;

public class ABC_156_C {
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);

        final int N = sc.nextInt();

        final int[] X = new int[N];
        for (int i = 0; i < N; i++) {
            X[i] = Integer.parseInt(sc.next());
        }

        int ans = Integer.MAX_VALUE;
        for (int i = 0; i < 100; i++) {
            double total = 0;
            for (int j = 0; j < N; j++) {
                total += Math.pow(X[j] - (i+1), 2);
            }
            ans = Math.min(ans, (int)total);
        }

        System.out.println(ans);
    }
}

File: kotlin/src/abc/abc156/ABC156_C.kt
package abc.abc156

import java.util.*
import kotlin.math.floor

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)

    val a = sc.nextInt()!!
    val b = sc.nextInt()!!

    (1..1000).forEach { i ->
        if (a.toDouble().equals(floor(i * 0.08)) && b.toDouble().equals(floor(i * 0.1))) {
            println(i)
            return
        }
    }

    println(-1)
}